pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -532,10 +541,10 @@ func (pl *InterPodAffinity) getMatchingAntiAffinityTopologyPairsOfPods(pod *v1.P
 // satisfiesPodsAffinityAntiAffinity checks if scheduling the pod onto this node would break any term of this pod.
 func (pl *InterPodAffinity) satisfiesPodsAffinityAntiAffinity(pod *v1.Pod,
 	state *preFilterState, nodeInfo *nodeinfo.NodeInfo,
-	affinity *v1.Affinity) (predicates.PredicateFailureReason, error) {
+	affinity *v1.Affinity) (string, error) {
```

Comment:
return two booleans instead of strings, one to indicate if affinityRulesNotMatch and the other to indicates antiAffinityRulesNotMatch.

Reply:
Got it, let me update